packages/wallet-specs/package.json

```diff
@@ -8,9 +8,10 @@
   "license": "MIT",
   "scripts": {
     "prepare": "yarn build",
-    "build": "npx tsc && yarn generateConfigs",
+    "build": "npx tsc -b",
```

Comment:
Is there a reason we don't want to generate configs on build? Speed?

Reply:
👍 They aren't checked into the repository right now.